ansible/roles/data-products-deploy/defaults/main.yml

```diff
@@ -259,7 +259,7 @@ uci_postgres:
   user_identities_table_name: "identities"
 
 uci_encryption_secret_key: "{{uci_encryption_key_base64}}"
-uci_pdata_id: "{{env}}.uci.{{sunbird_instance}}"
+uci_pdata_id: "{{uci_env}}.uci.{{sunbird_instance}}"
```

Comment:
Why the new variable? we have already predefined variables for the env specific. Could you please reuse those? if possible you can check with DevOps.

Reply:
![WhatsApp Image 2022-03-28 at 12 04 46](https://user-images.githubusercontent.com/25149840/160989183-441e7b16-2388-4ad9-9c14-ba877dbaf000.jpeg)

Please see this screenshot, Keshav suggested to add this new environment, so as to not have this issue occurring again and again.